content/docs/services/redis.md

```diff
@@ -34,14 +34,45 @@ During platform maintenance each Redis instance in the cluster will be temporari
 
 To ensure your application stays up during this process, configure your client to use a short (~ 1 second) timeout and to automatically reconnect to Redis when disconnected.
 
+## Managing backups
 
-## Rotating credentials
+The Redis service does not expose access to files generated by the Redis [SAVE](https://redis.io/commands/save) command. To back up data from your Redis instance, you can iterate over keys, fetch their values and expiration times using [DUMP](https://redis.io/commands/dump) and [TTL](https://redis.io/commands/ttl), and save them to a file. To restore, use the [RESTORE](https://redis.io/commands/restore) command for each saved key.
 
-You can rotate credentials by creating a new instance and deleting the existing instance. If this is not an option, email [cloud.gov support](mailto:cloud-gov-support@gsa.gov) to request rotating the credentials manually.
+### Connecting to your Redis service locally
 
-## Managing backups
+To facilitate backing up your Redis service, cloud.gov offers port forwarding if
+SSH is enabled for your organization and application. Follow the following
+instructions to create a service key and use it to set up a SSH tunnel.
 
-The Redis service does not expose access to files generated by the Redis [SAVE](https://redis.io/commands/save) command. To back up data from your Redis instance, you can iterate over keys, fetch their values and expiration times using [DUMP](https://redis.io/commands/dump) and [TTL](https://redis.io/commands/ttl), and save them to a file. To restore, use the [RESTORE](https://redis.io/commands/restore) command for each saved key.
+Create a [service key](https://docs.cloudfoundry.org/devguide/services/service-keys.html)
+to access Redis credentials:
+
+```sh
+cf create-service-key my-redis-backup my-key
+```
+
+Connect to your Redis service using port forwarding. Note: You'll need to
+leave the [`cf ssh`]({{< relref "docs/apps/using-ssh.md" >}}) command running
+and follow the next steps in a different terminal so that you can access the
+remote Redis instance from your local environment:
+
+```sh
+redis_credentials=$(cf service-key my-redis my-key | tail -n +3)
+
+redis_hostname=$(echo "${redis_credentials}" | jq -r '.hostname')
+redis_port=$(echo "${redis_credentials}" | jq -r '.port')
+redis_username=$(echo "${redis_credentials}" | jq -r '.username')
+redis_password=$(echo "${redis_credentials}" | jq -r '.password')
+
+cf ssh my-app -L "9200:${redis_hostname}:${redis_port}"
```

Comment:
Just a nit, but wdyt about using the default redis port (6379) instead of the default elastic port (9200)?

Reply:
yeah sounds great! I can update the PR with that change then you can approved.